README.md

```diff
@@ -65,7 +68,7 @@ What does the following selector do?  `ul.dropdown > li`?
 Select 1:
 ```
 [] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[ x ] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
 [] Selects all ul's of class dropdown, as well as the children elements that are li's
 [] Selects all ul's of class dropdown, only if their children are exclusively li's
 ```
```

Comment:
The Answer to Question 5 is actually the first: `[X] Selects all li's which are directly inside a ul of class dropdown (children).`

It's selecting all children `<li>` of the `<ul>` class of "dropdown"